content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -152,13 +169,6 @@ volume types are allowed:
  * downwardAPI
  * emptyDir
 
-To guarantee that the pod can read the files of such volumes, volumes are
-created as if you specified `.spec.securityContext.fsGroup` as `0` for the Pod.
-If it is specified to a different value, this other value will of course be
-honored instead.
+## {{% heading "whatsnext" %}}
 
-As a by-product of this, folders and files for these volumes will have
-permissions for the group, even if `defaultMode` or `mode` to specific items of
-the volumes were specified without permissions to groups. For example, it is not
-possible to mount these volumes in a way that its files have permissions only
-for the owner.
```

Comment:
Why removing the above two paragraphs?

Reply:
See the commit description, that explains it:

```
Also, with the new implementation, the fsGroup was dropped. So removed
the mention to those limitations.
```